modules/nats/tests/test_nats.py

```diff
@@ -0,0 +1,101 @@
+from testcontainers.nats import NatsContainer
+from uuid import uuid4
+import pytest
+
+
+"""
+If you are developing this and you want to test more advanced scenarios using a client
+Activate your poetry shell.
+pip install nats-py
+This will get nats-py into your environment but keep it out of the project
+
+
+"""
+
+
+NO_NATS_CLIENT = True
```

Comment:
This section will check if you have the client available, if so then certain tests are available. If not then those tests are skipped.

Reply:
there's really no reason to exclude tests due to missing deps - just dont force a dep on the client code which is using the testcontainer

Reply:
What's the correct way to indicate that with poetry?

Reply:
> What's the correct way to indicate that with poetry?

extremely painfull :smile: